Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -128,7 +128,6 @@
 
 - type: alert
   id: Handcuffed
-  onClick: !type:RemoveCuffs  { }
```

Comment:
This should not be removed. Make it so both this and the verb uses the action instead of calling `TryUncuff`. Right now the verb bypasses the cooldown since it's not using the action.

Reply:
Thanks for the heads-up about the menu verb, missed that one.

My initial attempt at this PR was to do as you say and have `RemoveCuffs` perform the Action, however I could not find a satisfying way to call the `PerformAction` function - as noted in the starter message other alerts/verbs that would benefit from this kind of functionality do the action without triggering the Action button, bypassing the timer but slowing it down in other ways (for internals, it's by using a doafter). 

If you have advice on how to best implement calling an Action via an Alert, I'm happy to hear it.

Reply:
[This was discussed on the Discord](https://discord.com/channels/310555209753690112/770682801607278632/1206735960197562428) and pending a larger rework of the alert system as a whole, this PR will pivot to not using Actions to cooldown the uncuff action (currently looking into UseDelay).